docs/readthedocs/source/doc/Nano/Overview/install.md

```diff
@@ -10,15 +10,20 @@ For PyTorch Users, you can install bigdl-nano along with some dependencies speci
 ```bash
 conda create -n env
 conda activate env
-pip install bigdl-nano[pytorch]
+pip install --pre --upgrade bigdl-nano[pytorch]
 ```
 
 For TensorFlow users, you can install bigdl-nano along with some dependencies specific to TensorFlow using the following commands.
 
 ```bash
 conda create -n env
 conda activate env
-pip install bigdl-nano[tensorflow]
+pip install --pre --upgrade bigdl-nano[tensorflow]
+```
+
+```eval_rst
+.. note::
+    Since bigdl-nano is still in the process of rapid iteration, we highly recommend that you install nightly build version through the above command to facilitate your use of the latest features, rather than installing the latest release version such as 2.1.0 by ``pip install bigdl-nano[pytorch]``.
```

Comment:
> rather than installing the latest release version such as 2.1.0 by ``pip install bigdl-nano[pytorch]``.

to

> For stable version, please refer to the document and installation guide [here](https://bigdl.readthedocs.io/en/v2.1.0/doc/Nano/Overview/nano.html).